### ISSUE TITLE
[Feature request] "Fix all $RULE" code action 

### ISSUE THREAD

**Issue by rassie** [charliermarsh/ruff-lsp#46]
This would probably require an additional option for `ruff` itself, but it'd be very cool to be able to apply fixes of a particular rule all at once, while being able to weed through the others. A typical example could be `UP007` -- there could be dozens in a single file and it's rather tedious to apply fixes for each separate case e.g. when upgrading from Python 3.8 to 3.11. Would be nice to have "Fix all", "Fix UP007" and "Fix all UP007 in file" to choose from.

**Comment by charliermarsh** [comment]
Yeah that's a nice suggestion. We could probably detect when you have more than one instance of a given error in a file, and only expose it in those cases.
